x/project/handler.go

```diff
@@ -377,44 +379,44 @@ func handleMsgWithdrawFunds(ctx sdk.Context, k Keeper, bk bank.Keeper,
 		),
 	})
 
-	return sdk.Result{Events: ctx.EventManager().Events()}
+	return &sdk.Result{Events: ctx.EventManager().Events()}, nil
 }
 
 func payoutAndRecon(ctx sdk.Context, k Keeper, bk bank.Keeper, projectDid did.Did,
-	fromAccountId InternalAccountID, recipientDid did.Did, amount sdk.Coin) sdk.Error {
+	fromAccountId InternalAccountID, recipientDid did.Did, amount sdk.Coin) (*sdk.Result, error) {
```

Comment:
Not sure why sdk.Result return value was added. It is always nil or empty, so might as well remove it.

Reply:
removed